cpu/esp32/Makefile.include

```diff
@@ -59,7 +59,7 @@ PSEUDOMODULES += esp_wifi_any
 
 export CPU ?= esp32
 export TARGET_ARCH ?= xtensa-esp32-elf
-export ESPTOOL ?= $(ESP32_SDK_DIR)/components/esptool_py/esptool/esptool.py
+export ESPTOOL ?= esptool.py
```

Comment:
My reasoning to completely remove `$(ESP32_SDK_DIR)/components/esptool_py/esptool/` is that `RESET` already uses only `esptool.py` without the path in the `ESP32_SDK_DIR`. So it is required to have `esptool.py` in your `PATH`.
The `esp8266` also uses `esptool.py`. And if it is coherent for both, I can have the same handling for the two architectures.

It may require a documentation update too.

Also a fallback with `$(or $(wildcard $(ESP32_SDK_DIR)/components/esptool_py/esptool/esptool.py),esptool.py)` could easily work for me, but `reset` still uses `esptool.py` directly.

It could require additional cleanups, I first just did the minimum to make it work in my configuration. (DOCKER, no toolchain).

Reply:
If you prefer, I can currently remove this change from this PR and change the way of handling it in a dedicated discussion.

Reply:
@cladmi I agree with you, that the goal should be to be able to flash the device without having  the ESP32 SDK installed. It wouldn't make sense to install the ESP32 SDK when `riotdocker` is used for compilation. Thus, I agree to remove the whole path.

Therefore, I have checked the documentation of ESP32 MCU.

If the `riotdocker` is used, everything should be clear. In section `Preparing the Enviroment` of chapter [RIOT Docker Toolchain (riotdocker) ](http://doc.riot-os.org/group__cpu__esp32.html#esp32_riot_docker_toolchain), `esptool.py` is documented as following:

The ESP flasher program <b>```esptool.py```</b> is available at [GitHub](https://github.com/espressif/esptool). Don't use the the ```esptool``` package of your OS. ```esptool.py``` requires either Python 2.7 or Python 3.4 or later. The latest stable version of ```esptool.py``` can be installed with ```pip```:
```
pip install esptool
```
If the user doesn't follow the documentation, it is his fault.

However, if the user installs the toolchain and follows the [Manual Toolchain Installation](http://doc.riot-os.org/group__cpu__esp32.html#esp32_manual_toolchain_installation), there is no word left about `esptool.py`. The reason is that it was supposed that `esptool.py` is used from `$(ESP32_SDK_DIR)/components/esptool_py/esptool/` in that case.

Therefore, we have to change the documentation and I would restructure it a bit once I'm touching it.

Reply:
@cladmi PR #12028 clarifies the installation of `esptool.py`, hopefully.

Reply:
Thanks I will look at it.